content/en/docs/tasks/observability/distributed-tracing/sampling/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Trace Sampling
+description: Learn the different approaches on how to configure trace sampling on the proxies.
+weight: 10
+keywords: [sampling,telemetry,tracing,opentelemetry]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Istio provides multiple ways to configure trace sampling. In this page you will learn and understand
+all the different ways sampling can be configured.
+
+## Before you begin
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
+
+## Available trace sampling configurations
+
+1.  Percentage Sampler: A random sampling rate for percentage of requests that will be selected for trace
+    generation.
+
+1.  Custom OpenTelemetry Sampler: A custom sampler implementation, that must be paired with the `OpenTelemetryTracingProvider`.
+
+### Percentage Sampler
+
+{{< boilerplate telemetry-tracing-tips >}}
+
+The random sampling rate percentage uses the specified percentage value to pick which requests to sample.
+
+The sampling rate should be in the range of 0.0 to 100.0 with a precision of 0.01.
+For example, to trace 5 requests out of every 10000, use 0.05 as the value here.
+
+There are three ways you can configure the random sampling rate:
+
+#### Globally via `MeshConfig`
+
+Random percentage sampling can be configured globally via `MeshConfig`.
+
+{{< text syntax=yaml snip_id=none >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    enableTracing: true
+    defaultConfig:
+      tracing:
+        sampling: 10
+{{< /text >}}
+
+#### Pod annotation `proxy.istio.io/config`
+
+You can add the `proxy.istio.io/config` annotation to your Pod metadata
+specification to override any mesh-wide sampling settings.
+
+For instance, to override the mesh-wide sampling above, you would add the following to your pod manifest:
+
+{{< text syntax=yaml snip_id=none >}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sleep
+spec:
+  ...
+  template:
+    metadata:
+      ...
+      annotations:
+        ...
+        proxy.istio.io/config: |
+          tracing:
+            sampling: 20
+    spec:
+      ...
+{{< /text >}}
+
+#### Telemetry API
+
+The random percentage sampler can also be configured via the Telemetry API.
+Via the Telemetry API, sampling can be configured on various scopes: mesh-wide, namespace or workload, offering great flexibility.
+To learn more, please see the [Telemetry API](/docs/tasks/observability/telemetry/) documentation.
+
+Here is an example to configure sampling via Telemetry API for the entire mesh:
+
+{{< text syntax=yaml snip_id=none >}}
+kubectl apply -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: mesh-default
+  namespace: istio-system
+spec:
+  meshConfig:
+    enableTracing: true
+    defaultConfig:
+      tracing:
+        sampling: 10
+EOF
+{{< /text >}}
```

Comment:
is this right?

Reply:
That is definitely not right, wrong snippet. Fixed now! [20f2aad](https://github.com/istio/istio.io/pull/14856/commits/20f2aadf048952c4df1ad67851af12e91f4d9903)